api/services/CalendarEvent/index.js

```diff
@@ -8,9 +8,21 @@ import slugify from 'slugify';
 
 export default class CalendarEvent extends BaseModelService {
   @saveInstance
-  create(body) {
+  async create(body) {
```

Comment:
Need to be inside a transaction. https://github.com/coding-blocks/gondor/blob/master/api/services/BaseModelService/index.js#L64

Reply:
Umm I don't get it.
Do you mean like this?

```
  @saveInstance
  @withTransaction
  async create(body, transaction) {
    let slug = slugify(body.title);
    const similarSlugsCount = await Models.CalendarEvent.count({
      where: {
        slug: {
          [Models.Sequelize.Op.startsWith]: slug,
        },
      },
      transaction,
    });
    .....
```

Reply:
yes, learn more here: https://sequelize.org/v5/class/lib/transaction.js~Transaction.html